src/App.tsx

```diff
@@ -0,0 +1,62 @@
+import logo from './logo.svg';
+import React, {useEffect, useState} from 'react'
+import Recipe from "./Recipe";
+import { Recipes } from "./types";
+import './App.css';
+
+const App = () => {
+  const APP_ID: string = "ba4b60a3";
+  const APP_KEY: string = "c75eafd8d65da34b433b93f3a7b47f6c";
+
+  let [recipes, setRecipes] = useState<Recipes[]>([]);
+  let [search, setSearch] = useState("");
+  let [query, setQuery] = useState("chicken");
+
+  useEffect(() => {
+    getRecipes();
+  }, [query]);
+
+  const getRecipes = async () => {
+    const response = await fetch(`https://api.edamam.com/search?q=${query}&app_id=${APP_ID}&app_key=${APP_KEY}`);
```

Comment:
you can specify return type here:
`const getRecipes = async (): Promise<void> => {`